ios-sdk.xcodeproj/project.pbxproj

```diff
@@ -779,7 +1084,6 @@
 		};
 		DB7A33F51C8F8E8B009CC743 /* Debug */ = {
 			isa = XCBuildConfiguration;
-			baseConfigurationReference = EEFF8429A821CE3BB0733D53 /* Pods-ios-sdkTests.debug.xcconfig */;
 			buildSettings = {
 				ALWAYS_EMBED_SWIFT_STANDARD_LIBRARIES = YES;
```

Comment:
Yeah these got removed here - https://github.com/mapzen/ios/commit/25038b51f6b9421da821bfc3c556c70181994c07

Looks like this isn't rebased on top of master. Lets do that so this stuff gets removed before merging this. Might be a fair bit of work given the changes involved.